NEWS.md

```diff
@@ -1,6 +1,41 @@
 # NEWS
 
-## 0.24
+## 0.24.1
+
+### Feature Management
+
+This release supports the upcoming Spec Version 0.0.4 that enables subgraph features to be declared in the manifest and
+validated during subgraph deployment
+[#2682](https://github.com/graphprotocol/graph-node/pull/2682)
+[#2746](https://github.com/graphprotocol/graph-node/pull/2746).
+
+> Subgraphs using previous versions are still supported and won't be affected by this change.
+
+### Api Version 0.0.5
+
+- Added better error message for null pointers in the runtime [#2780](https://github.com/graphprotocol/graph-node/pull/2780).
+
+### Environment Variables
+
+- Added `GRAPH_STORE_CONNECTION_MIN_IDLE` for setting the minimum number of allowed database connections.
+- Added `GRAPH_STORE_CONNECTION_IDLE_TIMEOUT` for setting the timeout of when a database connection is considered idle.
+- When `GETH_ETH_CALL_ERRORS_ENV` is unset, it doesn't make `eth_call` errors to be considered determinsistic anymore [#2784](https://github.com/graphprotocol/graph-node/pull/2784)
+
+### Robustness
+
+- Tolerate a non-primary shard being down during startup [#2727](https://github.com/graphprotocol/graph-node/pull/2727).
+- Check that at least one replica for each shard has a non-zero weight [#2749](https://github.com/graphprotocol/graph-node/pull/2749).
+- Reduce locking for the chain head listener [#2763](https://github.com/graphprotocol/graph-node/pull/2763).
+
+### Logs
+
+- Improve block ingestor error reporting for missing receipts [#2743](https://github.com/graphprotocol/graph-node/pull/2743).
+
+### Graph Manager (`graphman`)
+
+- Prevent migrations from running on setup [#2774](https://github.com/graphprotocol/graph-node/pull/2774).
```

Comment:
We don't need to mention this, because the thing that caused the issue never made it into a release.